datastore/schedule-export/main.py

```diff
@@ -0,0 +1,42 @@
+import base64
+import json
+import os
+
+from googleapiclient.discovery import build
+
+datastore = build('datastore', 'v1')
+project_id = os.environ.get('GCP_PROJECT')
+
+
+def datastore_export(event, context):
+    '''Trigger a Datastore export from a Cloud Scheduler job.
+    Args:
+         event (dict): event[data] must contain a json object encoded in
+         base-64. Cloud Scheduler encodes payloads in base-64 by default.
```

Comment:
NIT: I think you need additional indent here for continuous lines for argument description.